packages/application-components/src/components/tab-list/tab-list.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ReactNode } from 'react';
+
+export type TTabListProps = {
+  /**
+   * Any React node passed to TabHeader.
+   */
+  children: ReactNode;
+  /**
+   * Sets `aria-orientation` attribute. Defaults to `horizontal`
+   */
+  orientation?: 'horizontal' | 'vertical';
+};
+
+export const TabList = (props: TTabListProps) => {
+  return (
+    <div role="tablist" aria-orientation={props.orientation}>
```

Comment:
Would you consider all the logic with setting `aria-orientation` useful or should I hardcode it to `horizontal`?